feed-page/post/README.md

```diff
@@ -1 +1,57 @@
-# Post
\ No newline at end of file
+# Post
+
+```json
+{
+    "$type": "post@1", // Type definition with spec version number (1)
+    "id": 5, // Required, this id MUST be unique on the page this post is on. For example, this post could have the full id d448f1562c20dbafa42badd9f88560cd1adb2f177b30f0aa048cb243e55d37bd/feed/posts/1/5 (userId/structure/feedId/pageId/postId)
+
+    "repostOf":"5335179d69a3191ccb6329dcc0d2aaac2cada7ce145cbbd5543c8c2ee97e2a4a/feed/posts/0/11", // Optional, full ID of the post being reposted (If this key is present, this post is a repost and does not need to contain a "content")
+    "commentTo":"5335179d69a3191ccb6329dcc0d2aaac2cada7ce145cbbd5543c8c2ee97e2a4a/feed/posts/0/9", // Optional, full ID of the post this posts is commenting on
+    "parentHash":"sha256:24126b48452ce0243ae0f6bd3a3e0f9ee06ac01d5dab557c365b3e1bb8972fa5", // sha256sum of the JSON string of the post being reposted or commented on to prevent edits
+
+    "content": { // Every field in this map is OPTIONAL, if not specified otherwise
+        "mediaDuration": 1235312612, // duration of video or audio in milliseconds (int)
+        "video:": { // video content with different resolution and file formats
+            "mp4_noaudio+1920x1080": "sia://CACqf4NlIMlA0CCCieYGjpViPGyfyJ4v1x3bmuCKZX8FKA",
+            "mp4+1920x1080": "sia://CACqf4NlIMlA0CCCieYGjpViPGyfyJ4v1x3bmuCKZX8FKA/1920x1080_version.mp4",
+            "mp4+1280x720": "sia://CACqf4NlIMlA0CCCieYGjpViPGyfyJ4v1x3bmuCKZX8FKA/1280x720_version.mp4"
+        },
+        "audio": { // audio content with different resolution and file formats
+            "mp3": "sia://_A2zt5SKoqwnnZU4cBF8uBycSKULXMyeg1c5ZISBr2Q3dA"
+        },
+        "image": { // image content with different resolution and file formats
+            // also used as thumbnail when video is present
+            "png+1920x1080": "sia://IADUs8d9CQjUO34LmdaaNPK_STuZo24rpKVfYW3wPPM2uQ",
+            "png+1280x720": "sia://IADUs8d9CQjUO34LmdaaNPK_STuZo24rpKVfYW3wPPM2uQ",
+            "jpeg": "sia://IADUs8d9CQjUO34LmdaaNPK_STuZo24rpKVfYW3wPPM2uQ",
+        },
+        "text": "Short text or title", // REQUIRED, 512 Chars max, higlighted and used as title when description is available
+        "description": "", // Description, supports Markdown
```

Comment:
Instead of `description`, I think something like `content` could be better.
What do you think about adding an additional field `textContentType` to specify the content type of the content (HTML, markdown, plain text)?

Reply:
Changed :heavy_check_mark:

Reply:
Changed :heavy_check_mark: